ex4/ex4.js

```diff
@@ -6,3 +6,32 @@
  * toNumber("71") === 71
  * toNumber("a7") === null
  */
+function toNumber(str) {
```

Comment:
You've submitted ex4 by mistake in this pull request. Do you want to undo this, or just do a review for it here?